aks/job_configuration/variables.tf

```diff
@@ -0,0 +1,53 @@
+variable "namespace" {
+  description = "AKS namespace where this app is deployed"
+}
+variable "service_name" {
+  description = "Name of the service. Usually the same as the repo name"
+}
+variable "docker_image" {
+  description = "Docker image to be used for this job"
+}
+variable "environment" {
+  description = "Environment where this app is deployed. Usually test or production"
+}
+variable "azure_resource_prefix" {
```

Comment:
not in the module, probably the same for some of the other vars here.
check and remove those not required by the module